js/google_ima.js

```diff
@@ -1093,6 +1094,9 @@ require("../html5-common/js/utils/utils.js");
         var adErrorEvent = google.ima.AdErrorEvent.Type;
         _IMA_SDK_destroyAdsLoader();
         _IMAAdsLoader = new google.ima.AdsLoader(_IMAAdDisplayContainer);
+        // This will enable notifications whenever ad rules or VMAP ads are scheduled
+        // for playback, it has no effect on regular ads
+        _IMAAdsLoader.getSettings().setAutoPlayAdBreaks(false);
```

Comment:
We need to disable `autoPlayAdBreaks` even if we're not currently using ad rules because this code is only executed once per each set of videos and we might have to handle an ad rules tag in the future. This setting doesn't have any effect on regular ads.